next-app/src/lib/progress/types.d.ts

```diff
@@ -0,0 +1 @@
+type Id = string | number
```

Comment:
I found it preferable because it cleans up the files a bit with the true logic in them without losing the benefit of info from the editor when you hover.  I'll need to see how your workflow is different.  Thanks.

Reply:
migrated as discussed, see [20f57f6](https://github.com/sillsdev/web-languageforge/pull/1664/commits/20f57f65ea1c8837a0011a52dad5cac9363f7c5f)

Reply:
thank you, see [d7ecf3b](https://github.com/sillsdev/web-languageforge/pull/1664/commits/d7ecf3bf8030720f58f3322fb33b25a70925c38e)